doc/ruleSets/PhpCsFixer.rst

```diff
@@ -52,7 +52,12 @@ Rules
 - `phpdoc_var_annotation_correct_order <./../rules/phpdoc/phpdoc_var_annotation_correct_order.rst>`_
 - `return_assignment <./../rules/return_notation/return_assignment.rst>`_
 - `single_line_comment_style <./../rules/comment/single_line_comment_style.rst>`_
-- `single_line_throw <./../rules/function_notation/single_line_throw.rst>`_
 - `whitespace_after_comma_in_array <./../rules/array_notation/whitespace_after_comma_in_array.rst>`_
   config:
   ``['ensure_single_space' => true]``
+
+Disabled rules
+--------------
+
+- `phpdoc_to_comment <./../rules/phpdoc/phpdoc_to_comment.rst>`_
+- `single_line_throw <./../rules/function_notation/single_line_throw.rst>`_
```

Comment:
I don't understand - why this PR shows this as a change, here?

Reply:
This was a change from #6679 which this PR was based on and that has been merged since.